### ISSUE TITLE
major accessibility issues

### ISSUE THREAD

**Issue by zersiax** [FreeCodeCamp/COM1000#60]
Not sure if I'm doing something wrong, could be user error. Would appreciate thoughts.
When I go to the main page, seemingly triggered by the e/files express route, i get a list of files in my ~/freecodecamp/challenges. These files cannot be interacted with in any way I can see. Lastly in the DOM order is a 'choose files' button. Clicking this button seems to do nothing, according to screenreader. Nothing else appears to be on the page. 
Would like to add some accessibility and general information to some of the challenges, can't do that now since I don't really fancy editing the raw json data. Any comments very much appreciated :)

**Comment by benmcmahon100** [comment]
Hey @zersiax ! The menu is unfortunately made by an external library. I'm not sure how to fudge it to work. I believe that it's created on click, off the page, and slid into view. I wonder if the screen reader ignores it as it's not "visible" and then doesn't detect it's presence after the fact. As a weird question, is there any chance that you could take a screen shot of the page? It might be a proper error instead of a weird display issue
